go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/NethermindEth/juno v0.3.1
 	github.com/NethermindEth/starknet.go v0.7.1-0.20240401080518-34a506f3cfdb
 	github.com/XSAM/otelsql v0.27.0
-	github.com/avast/retry-go/v4 v4.6.0
+	github.com/avast/retry-go/v4 v4.5.1
```

Comment:
why did this downgrade? this is causing the build failure

Reply:
you should revert this change and run tidy

Reply:
fixed